README.md

```diff
@@ -1,3 +1,97 @@
+- `yarn start`: Starts the NextJS server in production mode
+- `yarn build:static`: Build the static site asset bundle for deploy
+- `yarn build:analyze`: Builds the asset bundle and generates webpack stats files
+
+### Working on an issue
+
+To begin working on an issue, make sure you've assigned yourself to the issue in Github and marked it as "In Progress.".
+
+Once you have an issue to work on, create a new branch off `main` using the naming convention:
+
+`{issue #}-{short description}`
+
+For example: `112-logo-component`
+
+We have a pre-commit hook set up using Husky to run on staged files only. This will run [Prettier](https://prettier.io/), [TypeScript compilation](https://www.typescriptlang.org/) and [eslint](https://eslint.org/) and fail on errors. For an optimal developer experience, it's recommended that you configure your editor to run linting & formatting inline.
+
+When your branch is ready, open a PR against `main`, fill out the description and request code reviews. The code must pass the same linting and formatting checks mentioned above, as well as [Jest tests](https://jestjs.io/) in order to merge.
+
+### PR Linting
+
+In order to easily maintain a clear changelog, we require all PRs into the `main` branch to pass the [conventional commits spec](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional).
+
+The format for PR commits is:
+
+```
+<type>(scope): <subject>
+<BLANK LINE>
+<body>
+<BLANK LINE>
+<footer>
+```
+
+#### Allowed `type`s are:
+
+- `build`: Changes that affect the build system or external dependencies (example scopes: webpack, npm)
+- `chore`: Completing a task that has no effective code changes, such as updating the version and changelog for a release
+- `ci`: Changes to our CI configuration files and scripts (example scopes: Circle, Github actions/workflows)
+- `docs`: Documentation only changes
+- `feat`: A new feature
+- `fix`: A bug fix
+- `perf`: A code change that improves performance
+- `refactor`: A code change that neither fixes a bug nor adds a feature
+- `revert`: If the commit reverts a previous commit, it should begin with `revert:` , followed by the header of the reverted commit. In the body it should say: `This reverts commit <hash>.`, where the hash is the SHA of the commit being reverted.
+- `style`: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
+- `test`: Adding missing tests or correcting existing tests
+
+#### `scope` is optional, and can be one of the following (this list is not exhaustive):
+
+- `deps`: Updating a package listed in dependencies
+- `deps-dev`: Updating a package listed in devDependencies
+- `ci`: Changes to Github Actions, CI config and/or scripts
+- `storybook`: Changes to Storybook or stories files only
+
+#### `body`:
+
+Will default to a list of included commits since PRs are all squashed when merging. You can also add additional context to the body if needed.
```

Comment:
Is it best practice to leave all these included commits, or is it okay to edit?

Reply:
IMO it is fine to edit, this is just the default behavior